kobo/hub/models.py

```diff
@@ -200,6 +204,8 @@ class Worker(models.Model):
     def __unicode__(self):
         return u"%s" % self.name
 
+    def __str__(self):
+        return u"%s" % self.name
```

Comment:
Should return `str` and not `unicode` on py2.

```suggestion
        return "%s" % self.name
```